__integrations__/index.test.ts

```diff
@@ -318,6 +336,57 @@ describe("Create Message", () => {
     }
   );
 });
+describe("Create Advanced Message", () => {
+  it.each`
+    profileType         | fiscalCode                       | serviceId
+    ${"LEGACY Profile"} | ${aLegacyInboxEnabledFiscalCode} | ${anEnabledServiceId}
+    ${"AUTO Profile"}   | ${anAutoFiscalCode}              | ${anEnabledServiceId}
+    ${"MANUAL Profile"} | ${aManualFiscalCode}             | ${anEnabledServiceId}
+  `(
+    "$profileType |> should return the message in PROCESSED status when service is allowed to send",
+    async ({ fiscalCode, serviceId }) => {
+      const body = {
+        message: {
+          fiscal_code: fiscalCode,
+          feature_type: "ADVANCED",
+          content: aMessageContent
+        }
+      };
+
+      const nodeFetch = getNodeFetch({
+        "x-subscription-id": serviceId,
+        "x-user-groups":
+          customHeaders["x-user-groups"] + ",ApiMessageWriteAdvanced"
+      });
+
+      const result = await postCreateMessage(nodeFetch)(body);
+
+      expect(result.status).toEqual(201);
+
+      const messageId = ((await result.json()) as CreatedMessage).id;
+      expect(messageId).not.toBeUndefined();
+
+      // Wait the process to complete
+      await delay(WAIT_MS);
+
+      const resultGet = await getSentMessage(nodeFetch)(fiscalCode, messageId);
+
+      expect(resultGet.status).toEqual(200);
+      const detail = (await resultGet.json()) as MessageResponseWithContent;
+
+      expect(detail).toEqual(
+        expect.objectContaining({
+          message: expect.objectContaining({
+            id: messageId,
+            ...body.message,
+            feature_type: "STANDARD"
```

Comment:
Am I wrong or should this be in ADVANCED feature type?

Reply:
message model is not yet implemented, this will be changed :D

Reply:
Ok, I'll review this when the new model will be merged.